src/js/index.js

```diff
@@ -76,29 +76,58 @@ class FHCookieGuard {
         });
     }
 
-    /**
-     * @private
-     */
     _onAcceptCookiesClick() {
-        this._setCookie(1);
+        this._setCookieConsentValue(this.options.cookieName, 1);
         this._enableCookieGuardedContent();
         this._closeCookieAlert();
     }
 
-    /**
-     * @private
-     */
     _onRefuseCookiesClick() {
-        this._setCookie(0);
+        this._setCookieConsentValue(this.options.cookieName, 0);
         this._closeCookieAlert();
     }
 
+    _initAutoAcceptCookieConsent() {
```

Comment:
Dit is geen "init" methode, cookies worden gelijk geaccepteerd in deze methode. Dus de naamgeving zou daar ook naar moeten zijn.
Bijvoorbeeld: `autoAcceptCookiesIfNeeded`